### PR TITLE
Fix #AR-3945 Readme updated as per Arcana Standard Simple Textual Rea…

### PR DESCRIPTION
**Pull request by shaloo**
…dme template

Ref: https://team-1624093970686.atlassian.net/browse/AR-3945 and https://team-1624093970686.atlassian.net/l/cp/6wNHommP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,74 @@
...
+It does not have any code for invoking other features and functions of these integrated SDKs.  Check out Arcana [Sample Demo App](https://github.com/arcana-network/demo-app) repository for a full fledged sample dApp that integrates with Arcana SDKs and plugs in Auth and Storage functionalities.
 
-### `npm start`
+This repository is bootstrapped using [Create React App](https://github.com/facebook/create-react-app).
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## ⚙️ Installation
+
+### Deploy Software Dependencies
```
Comment (shaloo) [inline README.md]:
```suggestion
### Download Software Dependencies

File: README.md [hunk 2]
```diff
@@ -1,35 +1,74 @@
...
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## ⚙️ Installation
+
+### Download Software Dependencies
+
+```bash
+npm install
+```
+
+### Run Development Server
```
Comment (shaloo) [inline README.md]:
```suggestion
### Run the Development Server

File: README.md [hunk 3]
```diff
@@ -1,35 +1,74 @@
...
+
+```bash
+npm test
+```
 
-Launches the test runner in the interactive watch mode.\
 See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run build`
+### Production Build
```
Comment (shaloo) [inline README.md]:
```suggestion
### Build for Production

File: README.md [hunk 4]
```diff
@@ -1,35 +1,74 @@
...
+This script builds the application for production in to the `build` folder.
 It correctly bundles React in production mode and optimizes the build for the best performance.
 
 The build is minified and the filenames include the hashes.\
 Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
-### `npm run eject`
+### Using Custom Configuration
```
Comment (shaloo) [inline README.md]:
```suggestion
### Use Custom Configuration

File: README.md [hunk 5]
```diff
@@ -39,32 +78,55 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
 
-### `npm run build` fails to minify
+#### Troubleshooting
+
+How do you address the case when the following script fails to minify?
+
+```bash
+npm run build
+```
+See [here](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify) for troubleshooting tips.
```
Comment (shaloo) [inline README.md:115]:
```suggestion

See [here](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify) for troubleshooting tips.

File: README.md [hunk 6]
```diff
@@ -39,32 +78,56 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+
+You can utilize the source code in this repository for getting started with Arcana Network SDKs. Use it to build a stand alone basic React application that integrates with the latest Auth and Storage SDK.  You may plug in additional functionality and try out various features of the SDKs.
+
+See [Arcana Documentation](https://docs.beta.arcana.network/) for user guides and SDK Reference Guides.  
+
+To get started with Arcana SDKs quickly, you can also check out other [code samples and templates](https://docs.beta.arcana.network/docs/overview_cs) in the `Developer Resources` section of the documentation.
+
+## 💡 Support
+
+For any queries regarding this repository usage, contact [Arcana support team](mailto:support@arcana.network).
```
Comment (shaloo) [inline README.md:127]: For any queries regarding this repository usage, contact the  [Arcana Support Team](mailto:support@arcana.network).

Comment (shaloo) [inline README.md]:
```suggestion
For any queries regarding this repository usage, contact the  [Arcana Support Team](mailto:support@arcana.network).

File: README.md [hunk 7]
```diff
@@ -39,32 +78,56 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+
+You can utilize the source code in this repository for getting started with Arcana Network SDKs. Use it to build a stand alone basic React application that integrates with the latest Auth and Storage SDK.  You may plug in additional functionality and try out various features of the SDKs.
+
+See [Arcana Documentation](https://docs.beta.arcana.network/) for user guides and SDK Reference Guides.  
+
+To get started with Arcana SDKs quickly, you can also check out other [code samples and templates](https://docs.beta.arcana.network/docs/overview_cs) in the `Developer Resources` section of the documentation.
+
+## 💡 Support
+
+For any queries regarding this repository usage, contact the  [Arcana Support Team](mailto:support@arcana.network).
```
Comment (shaloo) [inline README.md]:
```suggestion
For any queries regarding this repository usage, contact the [Arcana Support Team](mailto:support@arcana.network).

**Comment by mmjee** [comment]
Made some minor changes to fix grammatical errors. LGTM now.

**Pull request merged**